CustomEntitlementComputation

```diff
@@ -0,0 +1 @@
+Sources
```

Comment:
This symlink is necessary because SPM requires independent source directories. It's the same workaround we used for `ReceiptParser`.